utils/check_quickstart_uniqueness.ts

```diff
@@ -1,26 +1,24 @@
-import type { QuickstartConfig } from './types/QuickstartConfig';
-import type { FilePathAndContents } from './helpers';
-
+'use strict';
```

Comment:
We can remove this now that it's a typescript file, the compilation step handles setting strict mode for us!